cyarray/carray.pyx.mako

```diff
@@ -37,13 +38,11 @@ The numpy array may however be copied and used in any manner.
 
 # For malloc etc.
 from libc.stdlib cimport *
-IF UNAME_SYSNAME == "Windows":
-    cdef extern from "msstdint.h" nogil:
-        ctypedef unsigned int uintptr_t
-ELSE:
-    from libc.stdint cimport uintptr_t
+
+from libc.stdint cimport uintptr_t
 
 cimport numpy as np
+np.import_array();
```

Comment:
Please remove the semicolon.

Reply:
I have amended this and force pushed now.

Reply:
Thanks!